core/src/main/scala/org/ensime/config/EnsimeConfigProtocol.scala

```diff
@@ -28,17 +27,18 @@ object EnsimeConfigProtocol {
   private implicit val moduleFormat: SexpFormat[EnsimeModule] = cachedImplicit
   private implicit val configFormat: SexpFormat[EnsimeConfig] = cachedImplicit
 
-  def parse(config: String): EnsimeConfig = {
+  def parse(config: String, serverConfig: String): EnsimeConfig = {
```

Comment:
We could also use something like https://github.com/ccadllc/cedi-config but that adds another scala dependency